apps/zotonic_core/include/zotonic_file.hrl

```diff
@@ -16,65 +16,42 @@
 %% See the License for the specific language governing permissions and
 %% limitations under the License.
 
--record(z_file_info, {
-        modifiedUTC :: calendar:datetime(),
-        mime :: binary(),
-        size :: integer(),
-        acls :: list(integer()|{module, atom()}),
-        encodings :: list()
-    }).
-
--record(filestore, {
-        action = upload :: lookup | upload | delete,
-        path :: file:filename_all(),
-        filename :: file:filename_all() | undefined,
-        mime :: binary() | string() | undefined
-    }).
-
+-record(z_file_info,
+        {modifiedUTC :: calendar:datetime(),
+         mime :: binary(),
+         size :: integer(),
+         acls :: [integer() | {module, atom()}],
+         encodings :: list()}).
+-record(filestore,
+        {action = upload :: lookup | upload | delete,
+         path :: file:filename_all(),
+         filename :: file:filename_all() | undefined,
+         mime :: binary() | string() | undefined}).
 %%% @doc Notification to find the filestore credentials
--record(filestore_credentials_lookup, {
-        id :: integer(),
-        path :: binary()
-    }).
-
--record(filestore_credentials_revlookup, {
-        service :: binary(),
-        location :: binary()
-    }).
-
+-record(filestore_credentials_lookup, {id :: integer(), path :: binary()}).
```

Comment:
I know it fits on a single line,  wouldn't it be better (and consistent) to have it on multiple lines, like the other record definitions?

Reply:
You can add `-format #{inline_fields => none}.` to this module or the equivalent option to the `rebar.config` if you want it for all modules. But… that won't put `{` on the initial line.